coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -294,6 +298,17 @@ export default {
       })
     },
 
+    convertISOToCommonTime (isoTime) {
```

Comment:
We have not imported the 'moment' package yet, but if we start doing a lot with dates/times/formatting then we should consider it. I think we're not yet at that point, but just keep this in mind for later.